src/java/org/apache/cassandra/net/ParamType.java

```diff
@@ -40,25 +39,17 @@
  * will skip over any params it doesn't recognise.
  *
  * Please don't add boolean params here. Extend and use {@link MessageFlag} instead.
+ *
+ * Do not re-use old, of fill gaps in the sequence of, ids. New IDs must be higher.
```

Comment:
Nit: Shouldn't it be `or/nor` in `of fill gaps in...`?

Reply:
updated existing commit.